doc/DEMO.md

```diff
@@ -0,0 +1,629 @@
+
+watchtowers = [
+    "eecd2a93f5b09b88519f38d620aa127333d1934987d18001132f07ffa3596c65",
+    "614ad96890c309b8da6915ddb9eb1135caf228120178833e70798f20e0783b16",
+]
+```
+
+Now start the `coordinatord` again:
+
+```
+cd docker
+docker-compose up
+```
+
+Go back to the parent directory for the next step
+```
+cd ../..
+```
+
+### 8. Sending commands using the cli
+Let's go back to the revaultd directory and ask for info:
+```
+cd revaultd
+stake_1_cli getinfo
+```
+
+You should see something like:
+```
+{
+  "result":{
+    "blockheight":215,
+    "network":"regtest",
+    "sync":1.0,
+    "vaults":2,
+    "version":"0.0.2"
+  }
+}
+```
+
+Do the same with `stake_2_cli` and `man_1_cli`, just to make sure everything works :)
+
+### 9. Checking if you fucked up
+
+Please run
+
+```
+stake_1_cli getdepositaddress
+```
+
+and then
+
+```
+stake_2_cli getdepositaddress
+```
+
+and then
+
+```
+man_1_cli getdepositaddress
+```
+
+If they all return the same address, cool, somehow you managed to not fuck it up. Otherwise, well, you fucked up.
+Don't worrry, it happens. We're working towards making it happen less often by introducing descriptors in the configuration file, but in the meanwhile, you'll have to debug.
+
+If the deposit address are different, it means that the descriptors created by `revaultd` are different. Please check that:
+
+- The `stakeholder_xpubs` are all equal in all configurations, **the order of the keys MUST be the same**
+- The `manager_xpubs` are all equal in all configurations, **the order of the keys MUST be the same**
+- The `cosigner_keys` are all equal in all configurations, **the order of the keys MUST be the same**
+- The `unvault_csv` is the same in every configuration
+
+Delete your sqlite database and your watchonly wallet:
+
+```
+rm ./stake_1_data/revaultd.sqlite3
+rm ./stake_2_data/revaultd.sqlite3
+rm ./man_1_data/revaultd.sqlite3
+rm -r ./stake_1_data/revaultd-watchonly-wallet-1/
+rm -r ./stake_2_data/revaultd-watchonly-wallet-1/
+rm -r ./man_1_data/revaultd-watchonly-wallet-1/
+```
+
+Restart `revaultd` and check again the deposit addresses. If that didn't work either, try restarting `bitcoind`.
+
+### 10. Wait, CLI only?!
+Nah. We also have a GUI. Check out [revault-gui](https://github.com/revault/revault-gui) :)
+
+## Organizing the perfect Revault party
+
+Want to try out Revault on multiple computers? You can follow this tutorial, but please be mindful that:
+1. You'll need to be on the same Bitcoin network - either one common regtest/signet, or testnet.
+2. you'll need to reach the `cosignerd` and the `coordinatord` from outside `localhost`. You'll have to make sure to not bind on localhost (eg use `listen="0.0.0.0:<port>"` to listen on all interfaces)
+
+## Testing
+For testing you may find useful the [regtest manager script](../contrib/regtest_manager.sh). You will need to tweak the configuration we created in step 4 a bit, as the RPC address of the bitcoin node spawn by the script is `127.0.0.1:9001`.
```

Comment:
And that you use the same listen port for the cosigners that i use in the script, too (10001)

Reply:
Ah damn! I'll use a different port for the cosigners then